storage/innobase/handler/ha_innodb.cc

```diff
@@ -21877,6 +21906,161 @@ static void update_innodb_redo_log_encrypt(THD *thd, SYS_VAR *var,
   ut_ad(0);
 }
 
+/** Check if it is a valid value of innodb_undo_log_encryption_validate.
+Undo log encryption can be turned ON only if online to keyring
+encryption is disabled.
+@param[in]	thd	thread handle
+@param[in]	var	pointer to system variable
+@param[out]	save	immediate result for update function
+@param[in]	value	incoming bool
+@return 0 for valid innodb_track_changed_pages */
+static int innodb_undo_log_encryption_validate(THD *thd, SYS_VAR *var,
+                                               void *save,
+                                               struct st_mysql_value *value) {
+  const char *undo_log_encryption_input;
+  char buff[4];
+  int len = sizeof(buff);
+
+  ut_a(save != nullptr);
```

Comment:
is it same other update functions? asserts could be debug and on release build, if they are nullptr, we can return 1.